Day2/src/main/java/adventofcode2021/Day2.java

```diff
@@ -4,68 +4,91 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.List;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-import java.util.stream.Collectors;
 
-public class Day2 {
-
-	private static final String RESOURCE_FILE = "src/main/resources/puzzleInput.txt";
-
-	public static void main(String[] args) throws IOException {
-
-		List<String> movementsArray = getPuzzleInput();
-		System.out.println(getFinalHorizontalDepthPosition(movementsArray));
-		System.out.println(getFinalHorizontalDepthPositionWithAim(movementsArray));
-		System.out.println("Finish");
-	}
-
-	private static int getFinalHorizontalDepthPositionWithAim(List<String> movementsArray) {
-
-		int horizontal = 0;
-		int depth = 0;
-		int aim = 0;
-
-		for (String movement : movementsArray) {
```

Comment:
Since the parsing of the movent of the string is the same you could move it into a class or something to make it easier to interact.